### PR TITLE
Changes to landing page for Tower adv lab

### PR DESCRIPTION
**Pull request by goetzrieger**
##### SUMMARY
Couple of changes to make the default and attendance landing pages show info for the Advanced Tower lab when "create_cluster: true" and "workshop_type: rhel"

@liquidat 

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- provisioner

##### ADDITIONAL INFORMATION
- Add blocks to the landing page templates to show a different text and link to the proper lab guide when deployed as clustered RHEL lab
- Added "create_cluster: false" to group_vars all.yml

### CODE REVIEW

File: provisioner/group_vars/all/all.yml [hunk 1]
```diff
@@ -39,3 +39,4 @@ valid_security_console_types:
   - splunk
   - qradar
 ibm_community_grid: true
+create_cluster: false
```
Comment (liquidat) [inline provisioner/group_vars/all/all.yml:42]: @goetzrieger Can you add a new line character here? Build was fine, should work after this.

Comment (goetzrieger) [reply provisioner/group_vars/all/all.yml]: Done

**Comment by liquidat** [comment]
recheck

**Comment by liquidat** [comment]
Only F5 fails, and only for one of multiple nodes. Looks like a fluke.
Zuul failing builds don't make sense either. So we merge.

**Pull request merged**